app/views/admin/services/_history.html.erb

```diff
@@ -4,7 +4,7 @@
       </div>
       <div class="row">
     <% end %>
-    <div class="col-md-4 data-set">
+    <div class="col-md-4 data-set" id="<%="version-#{set.version}"%>">
```

Comment:
I reckon some-one with more frontend skills would prefer we didn't add ids to elements just so we can easily assert on them in tests.  I don't know what the preferred alternative might be (I'd most likely reach for a complicated xpath expresssion, but there may be better approaches).  Might be worth floating this past someone else to check.